src/main/java/lotto/domain/lotto/LottoLine.java

```diff
@@ -22,11 +22,11 @@ public LottoLine(List<LottoNumber> lottoNumbers) {
         value = unDuplicatedLottoNumber;
     }
 
-    public Rank matchLottoNumbers(List<LottoNumber> lottoNumbers, LottoNumber bonusNumber,
-                                  List<LottoNumber> answerLottoNumbers) {
-        int matchCount = (int) lottoNumbers.stream().filter(answerLottoNumbers::contains).count();
-        boolean hasBonusNumber = answerLottoNumbers.contains(bonusNumber);
-        return Rank.check(matchCount, hasBonusNumber);
+    public Rank matchLottoNumbers(List<LottoNumber> lottoNumbers, WinningNumbers winningNumbers) {
+        List<LottoNumber> winningLottoNumbers = winningNumbers.getLastWinningLottoNumbers().getValues();
+        int matchCount = (int) lottoNumbers.stream().filter(winningLottoNumbers::contains).count();
```

Comment:
```suggestion
        int matchCount = (int) lottoNumbers.stream()
                .filter(winningLottoNumbers::contains)
                .count();
```

stream 구문은 한줄에 하나씩 쓰는 것이 좋습니다 :) 다른 곳도 같이 체크해주세요

Reply:
b45725c 여기에 반영하였습니다!